tests/persistence_tests/test_shamir_persistence.py

```diff
@@ -194,7 +194,11 @@ def enter_shares_with_restarts(debug: DebugLink) -> None:
             assert expected_text in layout.text_content()
             layout = recovery.enter_share(debug, share)
             remaining -= 1
-            expected_text = "You have entered"
+            expected_text = (
+                "entered"
```

Comment:
nit: can we use `TR.regexp(recovery__share_from_group_entered_template)` instead?